scripts/build-module.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/node
+
+'use strict'
+
+const path = require('path')
+const glob = require('glob')
+const fs = require('fs')
+
+const babel = require('babel-core')
+const outputFileSync = require('output-file-sync')
+
+let options = JSON.parse(require('fs').readFileSync('.babelrc', 'utf8'))
```

Comment:
Ah right. I was confused by the explicit mention of preset later. Can we use babelrc "env" option instead to activate it?